iac-extensions/kubernetes/src/main/java/org/sonar/iac/kubernetes/visitors/CommentLocationVisitor.java

```diff
@@ -91,10 +92,16 @@ public void visitComment(InputFileContext ctx, YamlTree tree) {
 
   private void processComment(InputFileContext ctx, Comment comment) {
     var matcher = CONTAINS_LINE_NUMBER.matcher(comment.value());
+    var rangeMatcher = CONTAINS_LINE_NUMBER_RANGE.matcher(comment.value());
```

Comment:
It's not so good to execute 2 similar regexes on the same text. 
Maybe it will be better to only have second Pattern and make `rangeEnd` part optional?